snapd-glib/snapd-snap.c

```diff
@@ -897,13 +957,13 @@ snapd_snap_get_version (SnapdSnap *self)
  * snapd_snap_get_website:
  * @snap: a #SnapdSnap.
  *
- * Get the website of the snap developer, e.g. "http://example.com".
+ * Get the websites of the snap developer, e.g. ["http://example.com", "https://test.com"].
  *
- * Returns: a website URL.
+ * Returns: an array of URLs for website.
  *
  * Since: 1.50
  */
-const gchar *
+const GPtrArray *
 snapd_snap_get_website (SnapdSnap *self)
```

Comment:
This is an API/ABI break - we need to keep the old function around, and add a `snapd_snap_get_website2` or `snapd_snap_get_website_urls`. Did the old function ever work?

Reply:
As much I know, the old function was used only by the Gnome Software, which I can fix with an update. It did work, so I think this is worth taking a risk, as I'll implement it in the KDE side after the change.

Reply:
I don't think that we should break the ABI. I recommend  to go with Robert's proposal.

Reply:
I am keeping the ABI compatibility for the C part only. As we've not used in C++ yet. What about that?

Reply:
If these methods weren't in the Qt code then no action is required there. It is worth keeping the naming consistent between the GObject and Qt API though.

Reply:
Okay, I'm creating a new function in Qt. One is just `website` and another `website_urls`.